src/script/interpreter.cpp

```diff
@@ -823,6 +824,30 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                     CScriptNum start(stacktop(-2), fRequireMinimal);
                     CScriptNum length(stacktop(-1), fRequireMinimal);
 
+                    if (opcode == OP_SUBSTR_LAZY) {
+                        if (start < 0)
+                            start = 0;
+
+                        if (length < 0)
+                            length = 0;
+
+                        if (start >= vch1.size()) {
+                            popstack(stack);
+                            popstack(stack);
+                            popstack(stack);
+                            stack.push_back(vchZero);
+                            break;
+                        }
+
+                        if (length > MAX_SCRIPT_ELEMENT_SIZE)
+                            length = MAX_SCRIPT_ELEMENT_SIZE;
+
+                        // start + length cannot overflow because of the restrictions immediately abo
```

Comment:
https://github.com/ElementsProject/elements/blob/alpha/src/script/interpreter.cpp#L741

We lost the last 2 characters a long time ago; let's re-add them